lib/forgery/forgery/lorem_ipsum.rb

```diff
@@ -28,7 +28,7 @@ def self.character(options={})
   end
 
   def self.characters(quantity=10, options={})
-    options.merge!(:random_limit => lorem_ipsum_characters.length-quantity) if quantity.is_a?(Fixnum)
+    options.merge!(:random_limit => lorem_ipsum_characters.length-quantity) if quantity.is_a?(0.class)
```

Comment:
I think you are right. Looking closer there is not benefit to use `0.class` instead of `Integer` in this case. 

The overall benefit of this solution is that prior Ruby 2.4 a `Bignum` is not an `Integer`. That means `0.class` keeps the old behavior for Bignum in older versions of Ruby, `is_a?(Integer)` changes the behavior.

But on the other hand in this example it doesn't really make a difference, because it is very unlikely that someone passes a quantity bigger than an `Fixnum` could hold.

That said I updated my PR